pages/notifications.vue

```diff
@@ -20,30 +26,95 @@ const tabs = $computed<CommonRouteTabOption[]>(() => [
     display: isHydrated.value ? t('tab.notifications_mention') : '',
   },
 ])
+
+const filter = $computed(() => {
+  if (!isHydrated.value)
+    return undefined
+
+  const rawFilter = route.params?.filter
+  if (!rawFilter)
+    return undefined
+
+  if (NOTIFICATION_TYPES.includes(Array.isArray(rawFilter) ? rawFilter[0] : rawFilter))
+    return rawFilter as mastodon.v1.NotificationType
+
+  return undefined
+})
+
+const icons = NOTIFICATION_TYPES.reduce((acc, name) => {
+  switch (name) {
+    case 'status':
+      acc[name] = 'i-ri:account-pin-circle-line'
+      break
+    case 'reblog':
+      acc[name] = 'i-ri:repeat-fill'
+      break
+    case 'follow':
+      acc[name] = 'i-ri:user-follow-line'
+      break
+    case 'follow_request':
+      acc[name] = 'i-ri:user-shared-line'
+      break
+    case 'favourite':
+      acc[name] = 'i-ri:heart-3-line'
+      break
+    case 'poll':
+      acc[name] = 'i-ri:chat-poll-line'
+      break
+    case 'update':
+      acc[name] = 'i-ri:edit-2-line'
+      break
+    case 'admin.report':
+      acc[name] = 'i-ri:flag-line'
+      break
+    case 'admin.sign_up':
+      acc[name] = 'i-ri:user-add-line'
+      break
+  }
+  return acc
+}, {} as Record<string, string>)
+
+const filterText = $computed(() => (`${t('tab.notifications_more_tooltip')}${filter ? `: ${t(`tab.notifications_${filter}`)}` : ''}`))
+
+const more = $computed<CommonRouteTabOption[]>(() => NOTIFICATION_TYPES.filter(type => type !== 'mention').map(
```

Comment:
If we can determine that currentUser is not an admin, we should add a condition to not show the two notification types that are for admins only. But that could also be for another PR.

Reply:
I'll create a separate task for this one... I cannot find it yet